docs/Concepts/8.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Research in Progress
+---
+
+The following features are currently experimental and under research and initial implementation:
+
+**Economic Spam resistance**:
+We aim to enable an incentivized spam protection technique to enhance `WAKU-RELAY` by using rate limiting nullifiers.
+More details on this can be found in [17/WAKU2-RLN-RELAY](https://rfc.vac.dev/spec/17).
+In this advanced method, peers are limited to a certain rate of messaging per epoch and an immediate financial penalty is enforced for spammers who break this rate.
+
+**Prevention of Denial of Service (DoS) and Node Incentivization**:
+Denial of service signifies the case where an adversarial node exhausts another node's service capacity (e.g., by making a large number of requests) and makes it unavailable to the rest of the system.
+DoS attack is to be mitigated through the accounting model as described in [18/WAKU2-SWAP](https://rfc.vac.dev/spec/18).
```

Comment:
```suggestion
```

We don't intend to use SWAP in the short/mid term.

Prevention of DoS can refer to https://github.com/vacp2p/research/issues/148

Incenttivization : mention RLN + service credentials, here's a good article: https://vac.dev/building-privacy-protecting-infrastructure